swagger/v1/swagger.json

```diff
@@ -942,6 +942,32 @@
         }
       }
     },
+    "/confirmation": {
+      "get": {
+        "summary": "Signs up a new user; creates the user.",
```

Comment:
Looks like this is taken from the registration spec. We might want to update the summary.